website/server/models/group.js

```diff
@@ -323,6 +323,71 @@ schema.statics.getGroups = async function getGroups (options = {}) {
   return groupsArray;
 };
 
+function _translateSystemMessages (group, user) {
+  let foundText = '';
+  let lang = user.preferences.language;
+  for (let i = 0; i < group.chat.length; i++) {
```

Comment:
let' use `chat.forEach` instead of `for (`

Reply:
OK